lib/sensors/low_power_current.cpp

```diff
@@ -0,0 +1,29 @@
+#include <low_power_current.hpp>
+
+namespace hyped::sensors {
+
+std::optional<LowPowerCurrent> LowPowerCurrent::create(core::ILogger &logger,
+                                                       io::II2c &i2c,
+                                                       std::uint8_t channel = kdefault_i2c_address)
```

Comment:
`const`

Reply:
also `kDefaultI2cAddress`

Reply:
The [google style guide](https://google.github.io/styleguide/cppguide.html) might help with standard practice/naming style.